tasks/pip.yml

```
@@ -20,13 +20,15 @@
   until: result is success
   notify: reload supervisor
 
-- name: Install Flask, Werkzeug and uWSGI
+- name: Copy requirements.txt to virtualenv dir
+  copy:
+    src: "../requirements.txt"
+    dest: "{{ virtualenv_path }}/requirements.txt"
+  register: requirements_copy
```

I am unsure about the destination of the requirements file. With the current changes, it will be stored at 
`/opt/wireguard-openwisp/env/requirements.txt`. It will be unusual to store the requirements file inside the virtual environment. 

```suggestion
- name: Copy requirements.txt 
  copy:
    src: requirements.txt
    dest: "{{ openwisp2_wireguard_path }}/requirements.txt"
```

---

`requirements.txt` is very command in the Python eco-system, so we can keep using that. 

With my suggested changes, the file will be copied to `/opt/wireguard-openwisp/requirements.txt`

---

Ok, thanks for clarifying.

---

Updated the pip install task to use the requirements file from the new location